README.md

```diff
@@ -1,289 +1,102 @@
 
-### Download chat client
+## Use SimpleX Chat
 
-#### Linux and MacOS
+[<img src="https://github.com/simplex-chat/.github/blob/master/profile/images/apple_store.svg" alt="iOS app" height="42">](https://apps.apple.com/us/app/simplex-chat/id1605771084)
+&nbsp;
+[![Android app](https://github.com/simplex-chat/.github/blob/master/profile/images/google_play.svg)](https://play.google.com/store/apps/details?id=chat.simplex.app)
+&nbsp;
+[<img src="https://github.com/simplex-chat/.github/blob/master/profile/images/f_droid.svg" alt="F-Droid" height="41">](https://app.simplex.chat)
+&nbsp;
+[<img src="https://github.com/simplex-chat/.github/blob/master/profile/images/testflight.png" alt="iOS TestFlight" height="41">](https://testflight.apple.com/join/DWuT2LQu)
+&nbsp;
+[<img src="https://github.com/simplex-chat/.github/blob/master/profile/images/apk_icon.png" alt="APK" height="41">](https://github.com/simplex-chat/website/raw/master/simplex.apk)
 
-To **install** or **update** `simplex-chat`, you should run the install script. To do that, use the following cURL or Wget command:
+- 🖲 Protects your messages and metadata - who you talk to and when.
+- 🔐 Double ratchet end-to-end encryption + 2 additional encryption layers.
+- 📱 Mobile apps for Android ([Google Play](https://play.google.com/store/apps/details?id=chat.simplex.app), [APK](https://github.com/simplex-chat/website/raw/master/simplex.apk)) and [iOS](https://apps.apple.com/us/app/simplex-chat/id1605771084).
+- 🚀 [TestFlight preview for iOS](https://testflight.apple.com/join/DWuT2LQu) with the new features 1-2 weeks earlier - **limited to 10,000 users**!
+- 🖥 Available as a terminal (console) app / CLI on Linux, MacOS, Windows.
 
-```sh
-curl -o- https://raw.githubusercontent.com/simplex-chat/simplex-chat/master/install.sh | bash
-```
+## Make a private connection
 
-```sh
-wget -qO- https://raw.githubusercontent.com/simplex-chat/simplex-chat/master/install.sh | bash
-```
-
-Once the chat client downloads, you can run it with `simplex-chat` command in your terminal.
+You need to share a link or scan a QR code (in person or during a video call) to make a connection and start messaging.
 
-Alternatively, you can manually download the chat binary for your system from the [latest stable release](https://github.com/simplex-chat/simplex-chat/releases) and make it executable as shown below.
+The channel through which you share the link does not have to be secure - it is enough that you can confirm who sent you the message and that your connection is established.
 
-```sh
-chmod +x <binary>
-mv <binary> ~/.local/bin/simplex-chat
-```
+<img src="https://github.com/simplex-chat/.github/blob/master/profile/images/conversation.png" alt="Make a private connection" width="594" height="360">
```

Comment:
it'd be great to update the latest screenshot to include group member avatars in chat view, but the picture's neat anyway